app/mailers/decidim/direct_verifications/import_mailer.rb

```diff
@@ -24,6 +24,23 @@ def finished_registration(user, instrumenter)
           )
         end
       end
+
+      def successful_revoke(user, instrumenter)
+        @handler = :direct_verifications
+        @organization = user.organization
+        @stats = Stats.new(
+          count: instrumenter.emails_count(:revoked),
+          registered: instrumenter.processed_count(:revoked),
+          errors: instrumenter.errors_count(:revoked)
+        )
+
+        with_user(user) do
+          mail(
+            to: user.email,
+            subject: I18n.t("decidim.direct_verifications.verification.admin.imports.mailer.subject")
+          )
+        end
+      end
```

Comment:
We might be able to use the same mailer method and view if we manage to abstract the I18n key needed in each case. That's likely the only thing that changes.